application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -355,4 +362,15 @@ public String getTagName() {
             return tagName;
         }
     }
+
+    enum TicketStatus {
```

Comment:
open to a alt suggestion, but i think conveys the idea. Can't go with `thread activity` as it's already existing.